recipe/meta.yaml

```diff
@@ -44,16 +43,18 @@ test:
     - flake8 --ignore=D203,W503,E203 src\\flake8 tests\\integration tests\\unit setup.py  # [win]
     # 2022/03/24: the pip test does not work at this time
```

Comment:
It seems pip check is working so you can remove this comment now.

Reply:
Decided to remove the comment. Thanks!